doc/rst/usingchapel/chplenv.rst

```diff
@@ -372,12 +372,14 @@ CHPL_MEM
    Optionally, the ``CHPL_MEM`` environment variable can be used to select
    a memory management layer.  Current options are:
 
-        ========= =======================================================
-        Value     Description
-        ========= =======================================================
-        cstdlib   use the standard C malloc/free commands
-        jemalloc  use Jason Evan's memory allocator
-        ========= =======================================================
+        =============== =======================================================
+        Value           Description
+        =============== =======================================================
+        cstdlib         use the standard C malloc/free commands
+        jemalloc        use included version of Jason Evan's memory allocator
+        jemalloc-system use a system install of jemalloc
```

Comment:
Is this still right? I thought it was now `CHPL_MEM=jemalloc` `CHPL_JEMALLOC=system`